locale/ja/index.md

```diff
@@ -11,8 +11,8 @@ labels:
   tagline-lts: 推奨版
   changelog: 変更履歴
   api: API ドキュメント
-  version-schedule-prompt: または、
-  version-schedule-prompt-link-text: LTSのリリーススケジュールをご覧ください
+  version-schedule-prompt: For information about supported releases, see the
+  version-schedule-prompt-link-text: release schedule
```

Comment:
@nodejs/nodejs-ja  Can someone provide translations for these phrases (which will be put together to form a sentence in the page)?